docs/guides/1236-package-manager.md

```diff
@@ -0,0 +1,87 @@
+---
+slug: /1236/package-manager
+displayed_sidebar: "0.2"
+---
+
+# Interacting with the package manager
+
+Dagger's package manager currently lies in the `dagger project update` section of the CLI. Its aim is to *Download and install dependencies*.
+
+There are currently 2 types of dependencies that you can update:
+
+- Dagger ones (`dagger.io`, `universe.dagger.io`)
+- External dependencies
+
+All updated packages will reside in your `cue.mod/pkg` folder at the root of your project:
+
+```console
+$ tree -d -L 2 cue.mod/pkg/
+cue.mod/pkg/
+├── dagger.io
+│   ├── cue.mod
+│   └── dagger
+└── universe.dagger.io
+    ├── alpine
+    ├── aws
+    ├── bash
+    ├── cue.mod
+    ├── docker
+    ├── examples
+    ├── git
+    ├── go
+    ├── netlify
+    ├── nginx
+    ├── powershell
+    ├── python
+    ├── x
+    └── yarn
+
+18 directories
+```
+
+## Updating Dagger dependencies
+
+`dagger.io` and `universe.dagger.io` dependencies are vendored inside your binary version. If you change your Dagger version and `update` your dependencies, the vendored packages will be updated to their version at vendoring time.
+
+This is the reason behind [pushing-plan-dependencies](./1225-pushing-plan-dependencies.md): `dagger do` only relies on the packages present inside `cue.mod`, and you better push them to avoid having to `dagger project update` in your CI and risk encountering a breaking change.
+
+To update Dagger packages, after the installation of your CLI, please do:
```

Comment:
the `please` word personally reads unnecessarily polite for a guide. Maybe replace for something more neutral?